### ISSUE TITLE
tab_settings.lua can't be translated

### ISSUE THREAD

**Issue by ngosang** [minetest/minetest#2171]
Filters and mipmap list items in dev. version can't be translated because there are comparisons with the English text.

These are the strings:
```lua
local filters = {
	{"No Filter,Bilinear Filter,Trilinear Filter"},
	{"", "bilinear_filter", "trilinear_filter"},
}

local mipmap = {
	{"No Mipmap,Mipmap,Mipmap + Aniso. Filter"},
	{"", "mip_map", "anisotropic_filter"},
}
```
And these are the comparisons:
```lua
	if fields["dd_filters"] == "No Filter" then
		core.setting_set("bilinear_filter", "false")
		core.setting_set("trilinear_filter", "false")
		ddhandled = true
	end
	if fields["dd_filters"] == "Bilinear Filter" then
		core.setting_set("bilinear_filter", "true")
		core.setting_set("trilinear_filter", "false")
		ddhandled = true
	end
```

**Comment by kilbith** [comment]
No, it's not comparisons but conditions and these are unrelated with the translation. Btw, the dropdown elements can't be translated afaik.

**Comment by Zeno-** [comment]
https://github.com/minetest/minetest/pull/2172

**Comment by Zeno-** [comment]
Fixed by commit a28055f8fd8b09d1256c1da3c174b427f7b0c514
